news/_posts/core-weekly/2022-09-26-coreweekly-38-2022.md

```diff
@@ -0,0 +1,186 @@
+* [#29690](https://github.com/PrestaShop/PrestaShop/pull/29690): Disable apache mpm_event module in the CI, by [@atomiix](https://github.com/atomiix)
+* [#29687](https://github.com/PrestaShop/PrestaShop/pull/29687): Functional tests - Fix 'Customer service tests' in FO and BO. Thank you [@nesrineabdmouleh](https://github.com/nesrineabdmouleh)
+* [#29662](https://github.com/PrestaShop/PrestaShop/pull/29662): Add functional Tests - User account log out. Thank you [@cfarhani06](https://github.com/cfarhani06)
+* [#29640](https://github.com/PrestaShop/PrestaShop/pull/29640): Functional tests - Refacto test BO - Invoices - Other Options, by [@khouloudbelguith](https://github.com/khouloudbelguith)
+* [#29479](https://github.com/PrestaShop/PrestaShop/pull/29479): Sanity tests - Add new test 'Delete product' V2. Thank you [@nesrineabdmouleh](https://github.com/nesrineabdmouleh)
+* [#29444](https://github.com/PrestaShop/PrestaShop/pull/29444): Add workflow for product V2 sanity tests, by [@boubkerbribri](https://github.com/boubkerbribri)
+
+
+## Code changes in the '1.7.8.x' branch
+
+
+### Back office
+* [#29553](https://github.com/PrestaShop/PrestaShop/pull/29553): Add min height on alert to center icons vertically. Thank you [@lartist](https://github.com/lartist)
+
+
+### Front office
+* [#29632](https://github.com/PrestaShop/PrestaShop/pull/29632): Select new invoice address automatically. Thank you [@lartist](https://github.com/lartist)
+
+
+### Tests
+* [#29698](https://github.com/PrestaShop/PrestaShop/pull/29698): Disable apache mpm_event module in the CI (cherry-pick), by [@atomiix](https://github.com/atomiix)
+
+
+## Code changes in modules, themes & tools
+
+
+### Auto Upgrade module
+* [#509](https://github.com/PrestaShop/autoupgrade/pull/509): Add domain in all trans calls in twig templates, by [@jolelievre](https://github.com/jolelievre)
+* [#505](https://github.com/PrestaShop/autoupgrade/pull/505): Hardcode translation domains, by [@matthieu-rolland](https://github.com/matthieu-rolland)
+* [#497](https://github.com/PrestaShop/autoupgrade/pull/497): Fix version compare check to use CoreUpgrader 8 with 8.0 version. Thank you [@FabienPapet](https://github.com/FabienPapet)
+
+
+### Changes in developer documentation sources
+* [#1511](https://github.com/PrestaShop/docs/pull/1511): Backport of PR1510. Thank you [@leemyongpakvn](https://github.com/leemyongpakvn)
+* [#1510](https://github.com/PrestaShop/docs/pull/1510): Remove a wrong link in democqrshooksusage module. Thank you [@leemyongpakvn](https://github.com/leemyongpakvn)
+* [#1506](https://github.com/PrestaShop/docs/pull/1506): Change referenced module name in facet search documentation. Thank you [@thomasnares](https://github.com/thomasnares)
+* [#1505](https://github.com/PrestaShop/docs/pull/1505): Fix indentation on admin-controllers / tabs (Backport 1394). Thank you [@thomasnares](https://github.com/thomasnares)
```

Comment:
Maybe we should explain who `@thomasnares` is 😄 readers might wonder what is all this?

Reply:
Good idea! I've added information about Thomas joining the project. Let me know what you think.